_scripts/goaws.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+container_id=$(docker ps -a | grep pafortin/goaws | cut -d' ' -f1)
+if [ -z "$container_id" ]; then
+  docker run -d --name goaws -p 4100:4100 pafortin/goaws
+else
+  is_up=$(docker ps -a | grep pafortin/goaws | grep Up | cut -d' ' -f1)
+  if [ -z "$is_up" ]; then
+    docker start $container_id
```

Comment:
@deeptibaghel i don't see a docker stop anywhere here. Are we able to handle `./pm2` kill ?

Reply:
at which point should it call "docker stop" , so that it runs only during ./pm2 kill ?